tools/src/launcher/Main.hs

```diff
@@ -100,6 +100,20 @@ type M a = (HasConfigurations, HasCompileInfo) => Log.LoggerNameBox IO a
 
 data Executable = EWallet | ENode | EUpdater
 
+-- Node, its args, node log
+data NodeData = NodeData
+    { _nodePath    :: FilePath
+    , _nodeArgs    :: [Text]
+    , _nodeLogPath :: Maybe FilePath }
+
+-- Updater Node, its args, node log, archive path
```

Comment:
Updater is not a node, it's just a separate executable (which is currently unused, btw).
Can you remove `Node` from field names and this comment?